argparse_test.go

```diff
@@ -511,13 +605,13 @@ func TestFileSimple1(t *testing.T) {
 		t.Errorf("Test %s failed with error: %s", t.Name(), err.Error())
 		return
 	}
-	defer file1.Close()
-
```

Comment:
Please leave 1 empty line between logical blocks for better readability. Too dense code is very hard to read

Reply:
ok